src/core/ProgressBar/ProgressBar.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { useStyles } from './styles';
+import LinearProgress from '@material-ui/core/LinearProgress';
+import { BaseProgressProps } from './base';
+
+interface ProgressBarProps extends BaseProgressProps {
+  value: number;
+  color: string;
```

Comment:
`value` and `color` are already present in the [default API](https://material-ui.com/api/linear-progress/#linearprogress-api) of LinearProgressBar thus there is no need to Omit it in the base and reinitialize its type here.

Reply:
This comment is still unaddressed

Reply:
Only keeping this interface and removing the base file works 👍🏻